docs/content/latest/api/ycql/type_jsonb.md

```diff
@@ -42,6 +42,14 @@ type_specification ::= { JSONB }
 - Values of text data types with correct format are convertible to `JSONB`.
 - `JSONB` value format supports text literals which are valid json.
 
+{{< note title="Note" >}}
+
+Due to internal conversions, small float/decimal values are stored and shown with a significant error.
```

Comment:
We need better wording that this to explain the issue.

Reply:
Perhaps we can say something along the lines of 

```Internally, numbers that appear in a JSONB string (used without quotes. e.g `{ 'a': 5 } ` ) are stored as floating point values. Due to the inherent imprecision in storing floating-point numbers, one should either use error bounds while querying for these values in order to perform the correct floating-point comparision (show example), or store them as strings (e.g: { 'a' : "5" }```

Reply:
tbd: provide an example of using error-bounds on these values.